content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,435 @@
+
+Clicking the :guilabel:`Article` field reveals a drop-down menu. At first, there is only one option
+in the drop-down menu titled `Help`, which Odoo provides by default. Select `Help` from the
+drop-down to choose this article.
+
+.. tip::
+   To create a new article, go to :menuselection:`Main Dashboard --> Knowledge`. In the
+   :guilabel:`Knowledge` app, hover the cursor next to the :guilabel:`Workspace` section header,
+   located in the left sidebar. Moving the cursor there reveals a hidden "plus sign" icon
+   (:guilabel:`+`).
+
+   Click the "plus sign" icon (:guilabel:`+`) to create a new article in the :guilabel:`Workspace`.
+   In the upper right corner of the page, click the :guilabel:`Share` button, and slide the
+   :guilabel:`Share to Web` toggle switch until it reads :guilabel:`Article Published`. It can then
+   be added to a *Helpdesk* team.
+
+Once an article has been created and assigned to a *Helpdesk* team, content can be added
+and organized through the :guilabel:`Knowledge` app.
+
+.. seealso::
+   - :doc:`Editing Knowledge articles </applications/productivity/knowledge/articles_editing>`
+
+Search articles from a Helpdesk ticket
+--------------------------------------
+
+When members of a *Helpdesk* team are trying to solve a ticket, they can search through
+the content in the :guilabel:`Knowledge` app for more information on the issue.
+
+To search :guilabel:`Knowledge` articles, open a :guilabel:`ticket` - either from the
+:guilabel:`Helpdesk` app dashboard, or by going to :menuselection:`Helpdesk app --> Tickets --> All
+Tickets`, and selecting a :guilabel:`Ticket` from the list.
+
+When a :guilabel:`Ticket` is selected, Odoo reveals that :guilabel:`Ticket's` detail form.
+
+Click the :guilabel:`Knowledge` *book* icon, located above the :guilabel:`Chatter` window. Clicking
+the :guilabel:`Knowledge` *book* icon opens a search window.
+
+.. image:: help_center/help-center-knowledge-search.png
+   :align: center
+   :alt: View of knowledge search window from a helpdesk ticket.
+
+.. tip::
+   :guilabel:`Knowledge` articles can also be searched by pressing :command:`Ctrl + K` to open
+   the command pallette, then typing :command:`?`, followed by the name of the desired article.
+
+When Odoo reveals the desired article, click it, or highlight the :guilabel:`Article` title, and
+press :command:`Enter`. This will open the :guilabel:`Article` in the :guilabel:`Knowledge`
+application.
+
+To open the :guilabel:`Article` in a new tab, press :command:`Ctrl + Enter`.
+
+.. tip::
+   If a more in-depth search is required, press :command:`Alt + B`. That will reveal a separate
+   page, in which a more detailed search can occur.
+
+Share articles to the Help Center
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order for a :guilabel:`Knowledge` article to be available to customers and website visitors, it
+has to be published.
+
+.. note::
+   Even though the `Help` article has been enabled on a team, Odoo will not share all the nested
```

Comment:
I had originally been looking at this as a "user input", for which it would be correct to use backticks (as per @samueljlieber), however, I do agree it makes more sense to be italicized in this case.